rofication/_util.py

```diff
@@ -1,8 +1,11 @@
 import os
-from collections import MutableSequence, Callable
 from subprocess import check_output
 from typing import Optional
 
+try:
+    from collections.abc import MutableSequence, Callable
+except ImportError: # Python < 3.10
+    from collections import MutableSequence, Callable
 
```

Comment:
Why do you need that try? They removed the alias, but
```
from collections.abc import MutableSequence, Callable
```
Works in Python <3.10 as well, so there's no need to try.

Reply:
Yes, you're right. I did a commit removing it. I should have realized it earlier. 😅